src/lib/env.ts

```diff
@@ -1,5 +1,12 @@
 import { ZodIssueCode, z } from 'zod'
 
+const tryParsingAsBool = (val: unknown): boolean | unknown => {
+  if (val === "true" || val === "1") return true;
+  if (val === "false" || val === "0") return false;
+  // If not parseable, return the original value that will be caught in the zod validation
+  return val;
```

Comment:
Even better:

```suggestion
  return ['true', 'yes', '1', 'on'].includes(val.toLowerCase())
```

Note: in practice, `val` will always be a string, so no need to check for other types.